src/metabase/api/search.clj

```diff
@@ -515,12 +516,20 @@
    table_db_id  (s/maybe su/IntGreaterThanZero)
    models       (s/maybe models-schema)}
   (api/check-valid-page-params mw.offset-paging/*limit* mw.offset-paging/*offset*)
-  (search (search-context
-           q
-           archived
-           table_db_id
-           models
-           mw.offset-paging/*limit*
-           mw.offset-paging/*offset*)))
+  (let [start-time (System/currentTimeMillis)
+        results    (search (search-context
+                            q
+                            archived
+                            table_db_id
+                            models
+                            mw.offset-paging/*limit*
+                            mw.offset-paging/*offset*))
+        duration   (- (System/currentTimeMillis) start-time)]
+    ;; Only track global searches
+    (when (and (nil? models)
+               (nil? table_db_id)
+               (not archived))
+      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id* {:runtime-milliseconds duration}))
```

Comment:
```suggestion
      (snowplow/track-event! ::snowplow/new-global-search-query api/*current-user-id* {:runtime-milliseconds duration}))
```

should we make it explicit?

Reply:
I don't think its necessary unless there's more than one snowplow event that needs tracking